docs/source/project_documentation/user_guide/model_definitions.md

```diff
@@ -24,7 +24,7 @@ root key is used to represent how components fit together as part of the system
 and what those components do as part of the system.
 
 To compose a system from individual `component`s, one specifies additional
-`models` in the system `model`'s `components` field.
+`model`s in the system `model`'s `component`s field.
```

Comment:
Similar to below, these aren't referencing the `model` field, but the model as a whole. And `components` was the correct field reference, it is plural within the definition. 

```suggestion
models in the system model's `components` field.
```